werkzeug/serving.py

```diff
@@ -219,6 +219,19 @@ def run_wsgi(self):
         self.environ = environ = self.make_environ()
         headers_set = []
         headers_sent = []
+        filters_applied = set()
+
+        # Replaces nonlocal for python 2.7
+        class nonlocal_(object):
```

Comment:
that seems overly complicated, especially with the staticmethod() below. why not just use `nonlocal_ = [lambda data: data]` here and then `nonlocal_[0]` to access/reassign?

Reply:
Sure. Will do in the next commit

Reply:
@ThiefMaster pushed the change as you suggested.